scala3/getting-started.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: singlepage-overview
+title: Getting Started with Scala 3
+---
+
+
+
+This page describes how to get started with Scala 3. There are two main approaches, depending on your needs.
+
+1. If you want to experiment with Scala 3 right away, you can [run it in your browser](#run-in-browser).
+1. If you want to build a Scala 3 project on your computer, you just need to [set up Scala 3 on your computer](#install-tools). After that you can start creating projects, such as [creating a “Hello, world” project with sbt](#hello-world-example).
+
+The sections below take you through the necessary steps.
+
+
+
+## <a name="run-in-browser"></a>Run Scala in your browser
+
+To start experimenting with Scala 3 right away, use <a href="https://scastie.scala-lang.org/?target=dotty" target="_blank">the “Scastie” web application in your browser</a>. Scastie is an online “playground” where you can experiment with Scala examples to see how things work.
+
+<!-- TODO: provide some examples here? or, it would be nice if we could preload a Scastie session with some examples. -->
+
+
+
+## <a name="install-tools"></a>Set up Scala 3 on your computer
+
+[sbt](https://www.scala-sbt.org) is a commonly-used Scala build tool. To use it to create a Scala 3 project on your computer, you just need to install:
+
+- The Java 8 or newer
+- sbt 1.4 or newer
+
+Once you have those two tools installed, you can jump right to the [“Creating a ‘Hello, world’ project”](#hello-world-example) section of this page.
+
+
+### If you don’t have those tools installed
+
+If you don’t already have Java and sbt installed on your system, and you’d like a simple way to install them — along with other popular tools in the Scala ecosystem — a great approach is to use a tool named [Coursier](https://get-coursier.io/docs/cli-overview).
+
+Coursier is an “artifact-fetching” tool written in Scala. It’s designed to automatically fetch dependencies for you and works in parallel, so the artifacts you need are downloaded as rapidly as possible.
+
+### Installing Coursier
+
+The <a href="https://get-coursier.io/docs/cli-installation" target="_blank">Coursier installation page</a> shows how to install Coursier and its `cs` command on macOS, Windows, and Linux systems. See that page for how to install Coursier, and then come back to this page.
```

Comment:
Maybe `come back to this guide`. so we don't repeat `page` so often.

Reply:
```suggestion
The <a href="https://get-coursier.io/docs/cli-installation" target="_blank">Coursier installation page</a> shows how to install Coursier and its `cs` command on macOS, Windows, and Linux systems. See that page for how to install Coursier, and then come back to this guide.
```